tests/unit/tests_high_five_parallel.cpp

```diff
@@ -35,7 +35,9 @@ struct MpiFixture {
 };
 
 BOOST_GLOBAL_FIXTURE(MpiFixture)
-
+#if BOOST_VERSION >= 105900
```

Comment:
Thanks @alkino, in the end I think we need to do something like this.
I would maybe create a `#define BOOST_GLOBAL_FIXTURE_END` containing the semicolon (or empty) (in tests_high_five.hpp?) and use it here.